app/models/track.rb

```diff
@@ -29,6 +29,10 @@ class Track < ApplicationRecord
   delegate :debugging_instructions, :representer_normalizations, to: :git
   delegate :content, :edit_url, to: :mentoring_notes, prefix: :mentoring_notes
 
+  after_commit do
+    recache_num_exercises! if saved_changes.key?("course")
```

Comment:
Changing a track's `course` status should now update the number of exercises available.

Reply:
Nice.